bin/Analysis.cpp

```diff
@@ -151,6 +155,9 @@ void Analysis::createHistograms() {
 			<< endl;
 	lastNumberOfHistograms = numberOfHistograms;
 
+	pseudoTopAnalyser_->createTrees();
```

Comment:
These are now only created for central TTJets.  OK for now, but will need to be expanded to other samples for calculating the unfolding systematics.

To do this, I moved Analysis::createHistograms() down a few lines.  I don't think this matters, but it seems a more logical place to have that line?  Also, please check the method I use to check whether the current dataset is the TTJet sample.